search/django/weedid/tasks.py

```diff
@@ -31,9 +31,10 @@ def submit_upload_task(weedcoco_path, image_dir, upload_id):
             Path(weedcoco_path),
             Path(image_dir),
             Path(REPOSITORY_DIR),
-            Path(DOWNLOAD_DIR),
+            Path(TMP_DIR),
             upload_id,
         )
+        atomic_copy(Path(REPOSITORY_DIR), Path(DOWNLOAD_DIR), Path(TMP_DIR), upload_id)
```

Comment:
I think this can be included within deposit, rather than outside of it. We always want it whenever we deposit.